examples/bval-evaluation-redeployment/pom.xml

```diff
@@ -129,6 +129,20 @@
         <artifactId>cxf-rt-frontend-jaxrs</artifactId>
         <version>2.6.17</version>
         <scope>test</scope>
+        <exclusions>
+          <exclusion>
+            <groupId>javax.annotation</groupId>
+            <artifactId>javax.annotation-api</artifactId>
+          </exclusion>
+          <exclusion>
+            <groupId>javax.ws.rs</groupId>
+            <artifactId>javax.ws.rs-api</artifactId>
+          </exclusion>
+        </exclusions>
+        <exclusion>
```

Comment:
This seems to be out of the `<exclusions>` section. 
I get the following error when I tried to run the example:
```
mvn clean install
[INFO] Scanning for projects...
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] Malformed POM /Users/cesar/git/tomee/examples/bval-evaluation-redeployment/pom.xml: Unrecognised tag: 'exclusion' (position: START_TAG seen ...</exclusions>\n        <exclusion>... @142:20)  @ /Users/cesar/git/tomee/examples/bval-evaluation-redeployment/pom.xml, line 142, column 20
 @
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]
[ERROR]   The project org.superbiz:bval-evaluation-redeployment:8.0.0-SNAPSHOT (/Users/cesar/git/tomee/examples/bval-evaluation-redeployment/pom.xml) has 1 error
[ERROR]     Malformed POM /Users/cesar/git/tomee/examples/bval-evaluation-redeployment/pom.xml: Unrecognised tag: 'exclusion' (position: START_TAG seen ...</exclusions>\n        <exclusion>... @142:20)  @ /Users/cesar/git/tomee/examples/bval-evaluation-redeployment/pom.xml, line 142, column 20 -> [Help 2]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
[ERROR] [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/ModelParseException
``

Reply:
Resolved.  Forgot to push!